connector-j-6/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java

```diff
@@ -85,4 +81,17 @@ public Socket beforeHandshake() {
   public Socket afterHandshake() {
     return socket;
   }
+
+
+  // Returns True if running in a Google App Engine Standard runtime, otherwise False.
```

Comment:
```suggestion
  /** Returns {@code true} if running in a Google App Engine Standard runtime, otherwise {@code false}. */
```

I know it's private, but if you're going to document what the method does you might as well use Javadoc. Also, you could probably skip "otherwise false" since that's implied.

Reply:
Done.